src/sentry/api/endpoints/project_releases.py

```diff
@@ -37,13 +39,23 @@ def list_releases_scenario(runner):
     )
 
 
+class CommitSerializer(serializers.Serializer):
+    id = serializers.CharField(max_length=64)
+    repository = serializers.CharField(max_length=64, required=False)
```

Comment:
we simplify the default repository flow -- if you dont specify it all commits get lumped into one repo bound to the project name